### ISSUE TITLE
Warning when installing with zef: src/csvwrap.c:4:20: warning: type of ‘options’ defaults to ‘int’

### ISSUE THREAD

**Issue by demanuel** [tony-o/perl6-libcsv#2]
```
 $ zef install Text::CSV::LibCSV
===> Searching for: Text::CSV::LibCSV
===> Building: Text::CSV::LibCSV:ver<0.0.1>:auth<zef:tony-o>
[Text::CSV::LibCSV] src/csvwrap.c: In function ‘csv_create_parser_x’:
[Text::CSV::LibCSV] src/csvwrap.c:4:20: warning: type of ‘options’ defaults to ‘int’ [-Wimplicit-int]
[Text::CSV::LibCSV]     4 | struct csv_parser* csv_create_parser_x(options) {
[Text::CSV::LibCSV]       |                    ^~~~~~~~~~~~~~~~~~~
===> Building [OK] for Text::CSV::LibCSV:ver<0.0.1>:auth<zef:tony-o>
===> Testing: Text::CSV::LibCSV:ver<0.0.1>:auth<zef:tony-o>
===> Testing [OK] for Text::CSV::LibCSV:ver<0.0.1>:auth<zef:tony-o>
===> Installing: Text::CSV::LibCSV:ver<0.0.1>:auth<zef:tony-o>
```

I have libcsv 3.0.3

I have rakudo 2021.12-100-g596be5246

**Comment by tony-o** [comment]
@demanuel this issue should be resolved with [b703333](https://github.com/tony-o/perl6-libcsv/commit/b70333302773064bd801ffc486f8af5cecfac91a)

**Comment by demanuel** [comment]
@tony-o 

yes, i didn't have the issue anymore:

> zef install Text::CSV::LibCSV
===> Searching for: Text::CSV::LibCSV
===> Updating fez mirror: https://360.zef.pm/
===> Updated fez mirror: https://360.zef.pm/
===> Building: Text::CSV::LibCSV:ver<0.0.3>:auth<zef:tony-o>
===> Building [OK] for Text::CSV::LibCSV:ver<0.0.3>:auth<zef:tony-o>
===> Testing: Text::CSV::LibCSV:ver<0.0.3>:auth<zef:tony-o>
===> Testing [OK] for Text::CSV::LibCSV:ver<0.0.3>:auth<zef:tony-o>
===> Installing: Text::CSV::LibCSV:ver<0.0.3>:auth<zef:tony-o>
